src/components/com_tjcertificate/administrator/controllers/template.json.php

```diff
@@ -84,4 +85,27 @@ public function loadDefaultTemplate()
 			echo new JResponseJson($templateData);
 		}
 	}
+
+	/**
+	 * Function to load custom template
+	 *
+	 * @return  object  object
+	 */
+	public function loadCustomTemplate()
+	{
+		if (!JSession::checkToken('get'))
+		{
+			echo new JsonResponse(null, Text::_('JINVALID_TOKEN'), true);
+		}
+		else
+		{
+			$app   = Factory::getApplication();
+			$input = $app->input;
+			$templateId = $input->get('templateId');
+			$model = TJCERT::model('Template', array('ignore_request' => true));
+			$templateData = $model->getItem($templateId);
```

Comment:
use the template library here, and check first if  `$templateId` is empty

Reply:
done